lib/github-to-bitbucket-issues/downloaders/org_repository.rb

```diff
@@ -0,0 +1,10 @@
+module GTBI
+  module Downloaders
+    class Orgrepository < Base
```

Comment:
Can you change the class name to OrgRepository or the filename to orgrepository.rb? Which ever you prefer. :)